docs/guides/src/deploy/static_routing.md

```diff
@@ -77,7 +77,7 @@ addr = '127.0.0.1'
 port = 8831
 
 [[static_route.topology.schema_shards]]
-schema = 'public_0'
```

Comment:
Any difference between them?

Reply:
The config content differs from the description in the static routing section.

Reply:
Good catch